package.json

```diff
@@ -13,7 +13,7 @@
     "lint:js": "eslint \"**/*.js\" --ignore-pattern node_modules/",
     "lint:css": "sass-lint 'app/**/*.scss' -v -q",
     "nuke": "npm run clean && rm -rf node_modules/",
-    "start:dev": "babel-node server",
+    "start:dev": "cross-env LOGGER_LEVEL=info babel-node server",
```

Comment:
did you want to leave this is here?

Reply:
Not intentionally. But now that I see it. I quite like it so yeah let's leave it in there

Reply:
👍